assets/airports/lszh.json

```diff
@@ -0,0 +1,1386 @@
+        "ZH363": ["N47.865092", "E8.528153"],
+        "ZH365": ["N47.724570", "E8.554345"],
+        "ZH367": ["N47.543397", "E8.679225"],
+        "ZH368": ["N47.452758", "E8.741347"],
+        "ZH369": ["N47.362081", "E8.803256"],
+        "ZH370": ["N47.271408", "E8.864928"],
+        "ZH502": ["N47.465217", "E8.766339"],
+        "ZH503": ["N47.575000", "E8.709722"],
+        "ZH504": ["N47.456389", "E8.896945"],
+        "ZH506": ["N47.507222", "E8.780833"],
+        "ZH525": ["N47.440100", "E9.011075"],
+        "ZH526": ["N47.259269", "E8.620970"],
+        "ZH551": ["N47.302222", "E8.166667"],
+        "ZH552": ["N47.428889", "E8.391667"],
+        "ZH553": ["N47.422222", "E8.356111"],
+        "ZH554": ["N47.355086", "E8.248747"],
+        "ZH555": ["N47.346889", "E8.261267"],
+        "ZH556": ["N47.338333", "E8.384722"],
+        "ZH557": ["N47.313056", "E8.403611"],
+        "ZH558": ["N47.318056", "E8.144722"],
+        "ZH559": ["N47.517083", "E8.384667"],
+        "ZH570": ["N47.518000", "E8.505583"],
+        "ZH573": ["N47.534167", "E8.436667"],
+        "ZH578": ["N47.502695", "E8.459167"],
+        "ZH627": ["N47d22m20.7", "E8d37m13.7"],
+        "ZH628": ["N47d16m9.1", "E8d41m28.0"],
+        "ZH650": ["N47.362778", "E8.533333"],
+        "ZH676": ["N47.461111", "E8.504445"],
+        "ZH677": ["N47.577222", "E7.736945"],
+        "ZH701": ["N47.630833", "E8.667778"],
+        "ZH703": ["N47.485103", "E8.936397"],
+        "ZH713": ["N47.595306", "E8.337528"],
+        "ZH725": ["N47.253333", "E8.798056"],
+        "ZH726": ["N47.247222", "E8.624472"],
+        "ZH751": ["N47.556583", "E8.742889"],
+        "ZH775": ["N47.588333", "E8.128889"],
+        "ZUE"  : ["N47d35m31.00", "E8d49m3.00"]
+    },
+    "restricted": [
+        {
+            "name": "LS-R31",
+            "height": "8500ft",
+            "coordinates": [
+                ["N47d11m16", "E08d28m04"],
+                ["N47d10m09", "E08d30m00"],
+                ["N47d07m55", "E08d29m40"],
+                ["N47d02m15", "E08d23m11"],
+                ["N47d01m50", "E08d20m18"],
+                ["N47d03m54", "E08d19m19"],
+                ["N47d08m19", "E08d24m18"]
+            ]
+        }
+    ],
+    "runways": [
+        {
+            "name": ["10", "28"],
+            "end": [
+                ["N47.459000", "E8.537500", "1391ft"],
+                ["N47.456667", "E8.570500", "1416ft"]
+            ],
+            "ils": [false, true]
+        },
+        {
+            "name": ["14", "32"],
+            "end": [
+                ["N47.482167", "E8.536000", "1405ft"],
+                ["N47.461333", "E8.564500", "1402ft"]
+            ],
+            "ils": [true, false]
+        },
+        {
+            "name": ["16", "34"],
+            "end": [
+                ["N47.475667", "E8.536000", "1390ft"],
+                ["N47.449333", "E8.554167", "1385ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {

```

Comment:
Please add `ABLIX`, `GERSA` and `SONGI` departures. Even if they're rarely used, they should be added.

Reply:
added ALBIX, TRA.
I don't see the point in adding GERSA and SONGI, they're both only "TEMPORARY PROCEDURES" "ACTIVATION BY NOTAM OR BY AIRPORT AUTHORITY ONLY". They aren't even rarely used, they can't be used.

Reply:
I had never heard of a procedure activated by NOTAM, but fair enough, they can be omitted. We still need aircraft going to the south for departures to Italy so perhaps you could add a spawnPattern with the VEBIT departure and the T53 transition for these aircraft

Reply:
I added the T53 transition. The normal routing is a bit inconvenient because usually the aircraft are cleared direct gersa short after departure.

Reply:
Order SIDs alpha-numerically